app/Http/Controllers/Campaign/WebhookController.php

```diff
@@ -64,7 +64,7 @@ public function create(Campaign $campaign)
     {
         $this->authorize('webhooks', $campaign);
 
-        return view('campaigns.webhooks.create', ['campaign' => $campaign, 'model' => $campaign]);
+        return view('campaigns/webhooks.create', ['campaign' => $campaign, 'model' => $campaign]);
```

Comment:
view paths always use . Also, why is model set to campaign? That seems very wrong, it should be undefined